src/square.js

```diff
@@ -0,0 +1,10 @@
+export function Square({ color, onClick, onHover }) {
+  return (
+    <button
+      className="square"
+      onMouseDown={onClick}
+      onMouseOver={onHover}
+      style={{background: color}}
+    />
+  )
+}
```

Comment:
```suggestion
export default function Square({ color, ...rest }) {
  return (
    <button
      className="square"
      style={{background: color}}
      {...rest}
    />
  )
}
```

Reply:
Oh jk this won't work I don't think since you are passing it as `onClick` and not `onMouseDown`. The parent component would need to pass it as `onMouseDown` and `onMouseOver` instead of `onClick` and `onHover`. Up to you if you do that or not but going to leave the comment so you at least know it is an option. Pretty slick and helpful for redundant props